doc/test-requirements-template.md

```diff
@@ -11,9 +11,9 @@ assignees: ''
 Below is the required template for writing test requirements.  Good examples of test
 requirements include:
 
+* [TE-18.1 gRIBI MPLS in UDP Encapsulation and Decapsulation](https://github.com/openconfig/featureprofiles/blob/main/feature/gribi/otg_tests/mpls_in_udp/README.md)
 * [TE-3.7: Base Hierarchical NHG Update](/feature/gribi/otg_tests/base_hierarchical_nhg_update/README.md)
 * [gNMI-1.13: Telemetry: Optics Power and Bias Current](https://github.com/openconfig/featureprofiles/blob/main/feature/platform/tests/optics_power_and_bias_current_test/README.md)
-* [RT-5.1: Singleton Interface](https://github.com/openconfig/featureprofiles/blob/main/feature/interface/singleton/otg_tests/singleton_test/README.md)
```

Comment:
Any reason this isn't a good example any more? (Other than no canonical OC?)

Reply:
Yes, no canonical OC in that one and 2 examples seems like enough.